### ISSUE TITLE
Something is not working

### ISSUE THREAD

**Issue by gr2m** [ameerthehacker/probot-question-tagger-demo#5]
I expected xyz, but instead so this and that. Just testing the NLP app :)

**Comment by gr2m** [comment]
I expected this to be tagged as bug :)

**Comment by ameerthehacker** [comment]
Yes it should have been @gr2m but currently im supporting only detection of question, yet this still needs some rework. Just thought of sharing my initial steps to you
